### ISSUE TITLE
CSS minification

### ISSUE THREAD

**Issue by julianrubisch** [bullet-train-co/bullet_train#1428]
I was scrolling through the Lighthouse audit of my app when I noticed this:

<img width="649" alt="image" src="https://github.com/bullet-train-co/bullet_train/assets/4352208/88ff0721-0714-4171-bcc5-7f317eae68a6">

Though it's not a ton, it's easily fixed by just adding `--minify` here: https://github.com/bullet-train-co/bullet_train/blob/main/package.json#L39-L40

I wonder if it's worth upstreaming this to the starter repo? "pretty printing" the CSS in the browser's devtools still makes it easily inspectable.

Or we could add an environment variable, e.g. `MINIFY_CSS` to switch it on/off?

**Comment by jagthedrummer** [comment]
Oh, that's interesting. I do think we should figure out how to get `--minify` to be the default for production. Or maybe make it always the default and use an ENV var to skip minification? `SKIP_MINIFY_CSS` or something?

**Comment by julianrubisch** [comment]
Yep. Though to be honest, it seems the bulk of minification could be earned on the theme.
